ToolsQA/Tests/LoginTest.cs

```diff
@@ -1,63 +1,45 @@
 ﻿using NUnit.Framework;
 using OpenQA.Selenium;
-using System;
-using System.Collections.Generic;
 using System.Diagnostics;
-using System.Text;
 using ToolsQA.Pages;
 using ToolsQA.Selenium_Basics;
 
 namespace ToolsQA.Tests
 {
-    class LoginTest 
+    class LoginTest
     {
-        private string _username;
-        private string _password;
-        private string _url;
+        Cridentials cridentials = new Cridentials();
 
         [Test]
         public void CorrectLogin()
         {
-            _url = "https://pl.wikipedia.org/wiki/Wikipedia:Strona_g%C5%82%C3%B3wna";
-            _username = "";
-            _password = "";
-            Driver.StartBrowser();
-            Driver.OpenPage(_url);
+            cridentials._username = "Testowanie1";
+            cridentials._password = "Testy123";
```

Comment:
The above two lines should be encapsulated in other class like Users

class Users {
    public User TestUser() {
        return new User {
            Username = "",
            Password = ""
       }
    }
}